packages/website/docs/_components_pages/main/Input/Input.mdx

```diff
@@ -48,4 +49,9 @@ The sample demonstrates how the text of the suggestions wrap when too long.
 ### Custom Styling
 To some extent, the Input allows customization with pure CSS as shown in the sample.
 
-<CustomStyling />
\ No newline at end of file
+<CustomStyling />
+
+### Custom Icon Styling
+Custom icons added in the icon slot of the ui5-input can reuse the input icon styling with the help of our common css layer.
```

Comment:
Custom icons added in the icon slot of the `ui5-input` can reuse the input icon styling with the help of our Common CSS layer.